{"name": "Unsuccessfully login with email: simeon.hhl.qa@gmail.com and password: Test#123", "status": "passed", "steps": [{"name": "Check for error", "status": "passed", "start": 1732012844462, "stop": 1732012844537}, {"name": "Check the error text", "status": "passed", "start": 1732012844537, "stop": 1732012844547}], "parameters": [{"name": "email", "value": "'simeon.hhl.qa@gmail.com'"}, {"name": "password", "value": "'Test#123'"}, {"name": "error", "value": "'There was a problem logging you in. Please try again.'"}, {"name": "driver", "value": "'chrome'"}], "start": 1732012843843, "stop": 1732012844547, "uuid": "87e71e58-350c-4bae-b12a-326d74d40df9", "historyId": "13f6d2906b11515c4499357e2b32c203", "testCaseId": "c2811b1721d8231bbb0fb40778ae099c", "fullName": "Tests.test_Login.TestLogin#test_2", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "tag", "value": "flaky(reruns=3, reruns_delay=0.5, rerun_except='assert')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az665-524"}, {"name": "thread", "value": "1858-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}